pybliometrics/sciencedirect/__init__.py

```diff
@@ -8,4 +8,5 @@
 from pybliometrics.sciencedirect.object_retrieval import *
 from pybliometrics.sciencedirect.sciencedirect_search import *
 from pybliometrics.sciencedirect.subject_classifications import *
-from pybliometrics.scopus.serial_title import *
+from pybliometrics.scopus.serial_title_issn import *
+from pybliometrics.scopus.serial_title_issn import SerialTitleISSN as SerialTitle # For backward compatibility
```

Comment:
That's not good. Because then we forgo the opportunity to notify people of updating their code. We need to create a new module SerialTitle, that triggers a deprecation warning and at the same time imports SerialTitleISSN().